pydrofoil/real.py

```diff
@@ -0,0 +1,155 @@
+import sys
+from rpython.rlib.rbigint import rbigint, _divrem as bigint_divrem
+from rpython.rlib.rarithmetic import r_uint, intmask, string_to_int, ovfcheck, \
+        int_c_div, int_c_mod, r_ulonglong
+
+MAXINT = 2**63-1
+MININT = -2**63
+
+class Real(object):
+    def __init__(self, num, den, Normalize = True):
```

Comment:
Please rename the argument to `normalized`. I was unclear about this, but normalizing should be the default (because it's always safe to normalize, just inefficient), so the default value must be `False`. Also, default arguments don't take spaces around the name:

```python
    def __init__(self, num, den, normalized=False):
```